src/Runner.Worker/ActionManifestManager.cs

```diff
@@ -480,6 +480,10 @@ public ActionDefinitionData Load(IExecutionContext executionContext, string mani
                         return new CompositeActionExecutionData()
                         {
                             Steps = steps.Cast<Pipelines.ActionStep>().ToList(),
+                            PreSteps = new List<Pipelines.ActionStep>(),
+                            PostSteps = new Stack<Pipelines.ActionStep>(),
+                            InitCondition = "always()",
```

Comment:
let's brainstorm more

Reply:
chatted offline.... this just means the root pre/post step will run, but embedded pre/post conditions will still be respected

Reply:
👍

Reply:
why creating an empty list, just to initialize?